packages/core/src/Constants/Hints.ts

```diff
@@ -0,0 +1,48 @@
+import { type SignPage } from "./Titles";
+
+type HintData = {
+  header?: string;
+  description?: string;
+};
+const basicOperationHeader = undefined;
+const basicOperationDescription = undefined;
+
+const finalizationConsequences =
+  "Finalized funds are returned to your balance, allowing you to spend them or stake them again to earn rewards. Delegation rewards for these funds stop once they are finalized.";
```

Comment:
> Delegation rewards for these funds stop once they are finalized.

This is incorrect. Both spendable and unstaked balances are delegated. So moving from one to the other doesn't affect delegations.

```suggestion
  "Finalizing unstaked funds makes them available on your spendable balance. They can now be spent or staken again to earn more rewards.";
```